### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by khanh-devos**
- Created 2 new tables: species and owners.
- Added 2 columns owner_id & species_id into table animals referring to species & owners. 
- Added some data.
- Built some queries to solve the questions as below:

1. What animals belong to Melody Pond?
q1_join_tables_ON_owner_id

2. List of all animals whose type is Pokemon.
q2_join_tables_ON_species_id

3. List all owners and their animals, remember to include those that don't own any animal.
q3_left_join_tables

4. How many animals are there per species?
q4_join_group_by

5. List all Digimon owned by Jennifer Orwell.
q5_join_complex_ON

6. List all animals owned by Dean Winchester that haven't tried to escape.
q6_join_complex_ON

7. Who owns the most animals?
q7_join_max_max_ON

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,4 +10,43 @@ CREATE TABLE animals (
...
+    age INTEGER
+);
+
+-- Create a table species
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+CREATE TABLE SPECIES (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(100)
+)
```
Comment (usorfaitheloho) [inline schema.sql]: Kindly note that a table should have a primary key for the optimization of data from a database table so please add a primary key to your tables

File: queries.sql [hunk 2]
```diff
@@ -84,3 +84,76 @@ FROM (
...
+    INNER JOIN 
+    (SELECT id as i, full_name FROM owners WHERE full_name = 'Melody Pond') AS B 
+    ON A.owner_id = B.i;
+
+-- List of all animals that their type is Pokemon.
+SELECT id, name 
+    FROM animals AS A
+    JOIN
+    (SELECT id as i, name as n FROM species WHERE name = 'Pokemon') AS B
+    ON A.species_id = B.i;
```
Comment (usorfaitheloho) [inline queries.sql]: kindly note that the joins here ought to be LEFT JOIN and not INNER JOIN as seen in your project. Kindly implement these changes so as to have the right queries to be implemented 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @khanh-devos  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Database tables were created✔️
> * QUERIES commands were used✔️

> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (khanh-devos) [reply schema.sql]: Thank you, @usorfaitheloho,
I will set both id and name as primary keys.

File: queries.sql (hunk 2 above)
Comment (khanh-devos) [reply queries.sql]: Thank you, @usorfaitheloho ,
I will change it.

**Review by AbelG101 — APPROVED** [review]
# Hi @khanh-devos 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Tables are properly created & altered ✔
- Correct queries for all questions ✔
- Correct use of Git Flow ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**